### PR TITLE
Fix: lint/format fixes in several files

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
**What kind of change does this PR introduce?**

Formats the code files.

**Issue Number:**
#539 

**Screenshots/videos:**
N/A

**If relevant, did you update the documentation?**

N/A

**Summary**
Code files are now formatted

### CODE REVIEW

**Comment by frouioui** [comment]
There are some conflicts on this branch after merging #534

**Comment by DhairyaMajmudar** [comment]
Yes I have resolved that confilict.

File: website/src/pages/SearchPage/components/SearchMacro.tsx [hunk 1]
```diff
@@ -130,27 +109,51 @@ export default function SearchMacro({ data, gitRef }: SearchMacroProps) {
...
+  }
+=======
 function Row({ title, value, fmt }:RowProps) {
     var valFmt = value.center
     if (fmt == "time") {
         valFmt = secondToMicrosecond(value.center)
     } else if (fmt == "memory") {
         valFmt = formatByte(value.center)
     }
+>>>>>>> 85cc5af151fab94e3a4fbf022d03bf596ae36e8f
```
Comment (frouioui) [inline website/src/pages/SearchPage/components/SearchMacro.tsx:128]: Something bad has happened with a merge here, the frontend do not start with those.

**Review by frouioui — CHANGES_REQUESTED** [review]

File: website/src/pages/SearchPage/components/SearchMacro.tsx (hunk 1 above)
Comment (DhairyaMajmudar) [reply website/src/pages/SearchPage/components/SearchMacro.tsx:128]: Its running fine on my side 🤔 

image

Comment (frouioui) [reply website/src/pages/SearchPage/components/SearchMacro.tsx:128]: It should not, normally. But anyway these merge hints have to be removed

**Comment by DhairyaMajmudar** [comment]
I have opened a new PR to resolve all these conflicts.

Closing this one.

**Pull request closed without merging**